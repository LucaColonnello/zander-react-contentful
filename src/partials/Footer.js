import React from 'react';
import Nav from './Nav';
import NavSocial from './NavSocial';
import Bio from './Bio';

const Footer = () => (
  <footer role="contentinfo" className="footer">
    <div className="l-container">
      <Bio />
      <hr className="hr--short" />
      <Nav className="nav--footer" extras />
      <NavSocial />
    </div>
  </footer>
);

export default Footer;
