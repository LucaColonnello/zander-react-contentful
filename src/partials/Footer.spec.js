import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Footer from './Footer'

test('Footer snapshot test', () => {
  const component = shallow(
    <Footer />
  )
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
