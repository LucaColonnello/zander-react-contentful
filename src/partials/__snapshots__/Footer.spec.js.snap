// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test 1`] = `
<footer
  className="footer"
  role="contentinfo"
>
  <div
    className="l-container"
  >
    <Home />
    <hr
      className="hr--short"
    />
    <Nav
      className="nav--footer"
      extras={true}
    />
    <NavSocial />
  </div>
</footer>
`;
